typewise-alert.cpp

```diff
@@ -1,71 +1,52 @@
+#define ENABLE_BREACH_STATUS
+
 #include "typewise-alert.h"
-#include <stdio.h>
 
 BreachType inferBreach(double value, double lowerLimit, double upperLimit) {
+
+  BreachType retBreachType = NORMAL;
   if(value < lowerLimit) {
-    return TOO_LOW;
+    retBreachType = TOO_LOW;
   }
   if(value > upperLimit) {
-    return TOO_HIGH;
+    retBreachType = TOO_HIGH;
   }
-  return NORMAL;
+  return retBreachType;
 }
 
-BreachType classifyTemperatureBreach(
-    CoolingType coolingType, double temperatureInC) {
-  int lowerLimit = 0;
-  int upperLimit = 0;
-  switch(coolingType) {
-    case PASSIVE_COOLING:
-      lowerLimit = 0;
-      upperLimit = 35;
-      break;
-    case HI_ACTIVE_COOLING:
-      lowerLimit = 0;
-      upperLimit = 45;
-      break;
-    case MED_ACTIVE_COOLING:
-      lowerLimit = 0;
-      upperLimit = 40;
-      break;
-  }
-  return inferBreach(temperatureInC, lowerLimit, upperLimit);
+void classifyTemperatureBreach(std::unordered_map <int, std::pair<int, int>> &tempBreach) {
```

Comment:
Method name is not intuitive. ClassifytemperatureBreach gives an indication if the temperature is low, normal or high.
But in this case it is about configuring coolingtypes.